backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/nodes/AbstractDefinitionNodeDescriptionProvider.java

```diff
@@ -166,6 +167,7 @@ private NodePalette createNodePalette(NodeDescription nodeDescription, IViewDiag
                 .deleteTool(deleteTool.build())
                 .labelEditTool(editTool.build())
                 .edgeTools(edgeTools.toArray(EdgeTool[]::new))
+                .dropNodeTool(this.createDropFromDiagramTool(cache))
```

Comment:
are you sure about adding that new behavior to all node descriptions in every existing diagram descriptions?